invenio_rdm_pure/cli.py

```diff
@@ -28,6 +28,7 @@
 from docopt import docopt
 from shell_interface import ShellInterface, method_call
 from source.general_functions import check_if_directory_exists
+
```

Comment:
This file runs in a separate environment rather than the module itself.
we need to refactor the ```cli.py``` file for this module. and use the [Flask.commands](https://flask.palletsprojects.com/en/1.1.x/cli/).

Reply:
I think that should belong to another PR and be handled in #10.

I added my comment to this issue.